### ISSUE TITLE
Special case: required for radiobuttons

### ISSUE THREAD

**Issue by JAWS-test** [w3c/html-aam#339]
The specification says that required maps to aria-required. However, this is not always correct:
* in HTML, required is only allowed on the single radio button, but not on the fieldset.
* in ARIA aria-required is only allowed on radiogroup, but not on radiobutton

This should either be mentioned here. Alternatively, the ARIA or HTML specification should be modified to make required really equivalent to aria-required.

Related: https://github.com/whatwg/html/issues/6868, https://github.com/w3c/wcag/issues/1963 and https://github.com/w3c/aria/issues/1512

**Comment by scottaohara** [comment]
yes, the way that radio button groups are handled for their constraint validation is unique.  but the mapping is accurate for the attribute.  HTML already defines how this works, so I don't think HTML AAM needs to restate.
